src/suvela/ForLoop.java

```diff
@@ -0,0 +1,44 @@
+package suvela;
+
+public class ForLoop {
+
+	void even(int start, int end) {
+		for (int i = start; i <= end; i++) {
+			System.out.println("Even number is " + i);
```

Comment:
Here you should do if(i%2==0) right, consider a scenario where number starts with 11 then your code will not work as expected.